HW_5/task_3.py

```diff
@@ -0,0 +1,129 @@
+"""
+Напишите игру "Крестики-нолики".
+"""
+from random import randint
+import os
+
+
+def print_board(array: list):
+    print('|', end=' ')
+    print(*range(len(array)), sep=' | ', end=' | \n')
+    print('-' * 13)
+    for i, line in enumerate(array):
+        print('|', end='')
+        print(*line, sep='|', end=f'| {i} \n')
+        print('-' * 13)
```

Comment:
Что то у вас отображение съехало.
![image](https://user-images.githubusercontent.com/49741194/207253514-bc0b45cc-d735-4240-be84-51e8a97d0c02.png)
```suggestion
def print_board(array: list):
    print('|', end=' ')
    print(*range(len(array)), sep=' | ', end=' | \n')
    print('-' * 13)
    for i, line in enumerate(array):
        print('| ', end='')
        print(*line, sep=' | ', end=f' | {i} \n')
        print('-' * 13)
```